client/my-sites/upgrades/domain-management/transfer/unlocked.jsx

```diff
@@ -53,22 +48,35 @@ const Unlocked = React.createClass( {
 				this.setState( { submitting: false } );
 			}
 			if ( error ) {
-				notices.error(
-					this.translate(
-						'Oops! Something went wrong and your request could not be ' +
-						'processed. Please try again or {{a}}Contact Support{{/a}} if ' +
-						'you continue to have trouble.',
-						{
-							components: {
-								a: (
-									<a
-										href="https://support.wordpress.com/contact/"
-										target="_blank" />
-								)
-							}
-						}
-					)
-				);
+				const contactLink = <a href="https://support.wordpress.com/contact/" target="_blank"/>;
```

Comment:
As per #2513, it might be a good idea to change this link to https://wordpress.com/help/contact.